### DISCUSSION TITLE
Plugin rabbitmq_stream: How to set username and password?

### DISCUSSION THREAD

**Discussion by SimonWoidig** [rabbitmq/rabbitmq-server#11966] [Questions] [answered]
The documentation has nothing about setting the username and password. The user I have created, works for queues but not for streams. The documentation mentions the defaults being`guest:guest`.
Is there a way to change this? Thanks.

**Comment by mkuratczyk** [comment, marked as answer]
Authentication is not specific to the queue type / protocol used. If it "works for queues"  (I guess you mean over AMQP 0.9.1), it should also work over the Stream protocol. The default `guest:guest` user works for all protocols.

**Reply by michaelklishin** [reply]
I must mention that the default credentials only work for local connections [by design](https://www.rabbitmq.com/docs/access-control#loopback-users). What is recommended for remote ones can be read in the [Production Checklist guide](https://www.rabbitmq.com/docs/production-checklist).
